charts/base-cluster/templates/monitoring/security/trivy.yaml

```diff
@@ -44,11 +45,6 @@ spec:
         registry: {{ $.Values.global.imageRegistry }}
     {{- end }}
       ignoreUnfixed: true
-      resources:
-        requests:
-          memory: 256Mi
-        limits:
-          memory: 4Gi
```

Comment:
Removed?

Reply:
Probably remove accidentaly.

Will re-add it.

Reply:
Ah no, no accident.

I miss placed it before. According to the values, `resources` is a top level key of the values: https://github.com/aquasecurity/trivy/blob/main/helm/trivy/values.yaml#L21